src/ui/PrimitiveList.tsx

```diff
@@ -6,13 +6,22 @@ import {Primitive as LanguagePrimitive} from '../parsers/primitives';
 import {DragPrimitiveSource} from '../dnd';
 import {say} from '../utils';
 import SHARED from '../shared';
-import {copyNodes} from '../actions';
+import {copy} from '../actions';
```

Comment:
`copyNodes` is not a function that exists in actions... so I replaced it with `copy` which does exist. But I'm not sure this is correct.

Reply:
This is the correct behavior. :) Also fixed on master.